daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -351,6 +351,46 @@ async function start() {
 }
 ```
 
+#### Bulk Subscribe to messages
+
+Bulk Subscription is supported and is available through following API:
+
+- Bulk subscription through the `subscribeBulk` method: `maxMessagesCount` and `maxAwaitDurationMs` are optional; and if not provided, default values for related components will be used.
+
+Please refer [this document](https://v1-10.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/) for more details.
+
+```typescript
+import { DaprServer } from "@dapr/dapr";
+
+const pubSubName = "orderPubSub";
+const topic = "topicbulk";
+
+const DAPR_HOST = process.env.DAPR_HOST || "127.0.0.1";
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3502";
+const SERVER_HOST = process.env.SERVER_HOST || "127.0.0.1";
+const SERVER_PORT = process.env.APP_PORT || 5001;
+
+async function start() {
+  const server = new DaprServer(SERVER_HOST, SERVER_PORT, DAPR_HOST, DAPR_HTTP_PORT);
```

Comment:
TODO: Update this once #462 is merged.

Reply:
462 will take care of it, I plan to get this PR of Bulk Sub merged first.